docs/source/typed_dict.rst

```diff
@@ -289,6 +289,32 @@ need to give each TypedDict the same key where each value has a unique
 :ref:`Literal type <literal_types>`. Then, check that key to distinguish
 between your TypedDicts.
 
+You can also use ``@final`` to indicate that an instance of TypedDict will not be
+reassigned or subclassed. This allows mypy to do type refinement based on unique
```

Comment:
I'm not sure what reassigned means here?

Reply:
fixed, thanks!

official wording here fwiw: https://docs.python.org/3/library/typing.html#typing.final